### PR TITLE
upload images and svg index.html

### PR DESCRIPTION
**Pull request by KaterynaBabiak**

### CODE REVIEW

File: about-us.html [hunk 1]
```diff
@@ -90,233 +112,287 @@ <h1 class="hero-title">ABOUT US</h1>
...
+                </ol>
             </div>
         </section>
         <section class="our-team" id="our_team">
             <div class="container">
                <h2 class="section-title">OUR TEAM</h2> 
                <ul class="team-list">
                 <li class="team-item">
-                    <img src="./img/" alt="Ralph Edwards" class="team-img">
+                    <img src="./img/ralph-edwards-founder.webp" alt="" class="team-img">
```
Comment (per-aspera-ad-astra) [inline about-us.html]: Previous solution with text in `alt` attribute was better

**Comment by per-aspera-ad-astra** [comment]
Great job in general!
Some points to improvement:
- Some images have an empty alt attribute - better add text here.
- Name of image files - there are file names starting with a capital letter(for example Minus.svg), some starting with a lowercase letter, and there are spaces in the names (Icon Search.svg). There should be one approach to naming files - all words with a lowercase letter are separated by a hyphen (like most of your files).
- Please check the icons you load from the svg-sprite - I didn’t see any of them on my side. For example social media list:
Screenshot 2024-08-07 at 19 22 02

**Comment by KaterynaBabiak** [comment]
I've checked the way of adding svg from sprite and I  don't understand why you couldn't see all of them.  Could you check one more time, please?  And if it doesn't work could you explain me what i have to do to fix these issues? Because everything looks good on my screen.

team-section
join-us

**Comment by per-aspera-ad-astra** [comment]
Sorry for the inconvenience, you are right - the sprite icons works, it's my mistake.

There are one thing to improve that I missed the first time - I can't see `width` and `height `attributes on `img` tags - need to add them.

**Comment by KaterynaBabiak** [comment]
Thanks for comments, I've added some missing width & height property for img tags in styles.css.

**Comment by per-aspera-ad-astra** [comment]
> Thanks for comments, I've added some missing width & height property for img tags in styles.css.

I mean add width and height as html-attributes, not like css properties :)

**Comment by KaterynaBabiak** [comment]
Is it wrong way to add width and height as css properties?
I mean if we have a lot of images in one group with the same size, do we need to write same width and height next to each image?  
I will fixed all as you've recomended me but I just want to understand which way is better and why.

**Comment by per-aspera-ad-astra** [comment]
In the task description it is written:
"Add basic sizes: width and height, in tags." - I was guided by this point.

In real projects, a group of images with the same size is usually output using a loop, meaning you will need to specify the dimensions of the image once.

Your solution is not wrong - css is also used for define sizes.

You can read for example this article - [https://habr.com/ru/articles/524918/](https://habr.com/ru/articles/524918/)

**Comment by KaterynaBabiak** [comment]
Thanks for the explanation and the article was very useful for me.
I've changed the image sizes as it was written in the task description.

**Review by per-aspera-ad-astra — APPROVED** [review]

**Comment by per-aspera-ad-astra** [comment]
Approved.
Can be merged into main branch.

File: css/styles.css [hunk 2]
```diff
@@ -1878,6 +1884,136 @@ img {
...
+}
+
+.search-bar-subtitle {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: left;
+    text-transform: uppercase;
+    color: #FFFFFF;
```
Comment (per-aspera-ad-astra) [inline css/styles.css]: You can reduce` #FFFFFF` to `#fff`

File: css/styles.css [hunk 3]
```diff
@@ -1878,6 +1884,136 @@ img {
...
+.categories-list {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    padding: 12px 24px;
+}
+
+.categories-link {
+    font-size: 14px;
+    line-height: 22px;
```
Comment (per-aspera-ad-astra) [inline css/styles.css]: I would advise setting the `line-height` in dimensionless quantities - here it's 1.57 (22px / 14px). Would be good if you will do it whole project

File: contact-us.html [hunk 4]
```diff
@@ -120,33 +151,40 @@ <h2 class="get-in-title">
                     Get in touch
                 </h2>
                 <p class="get-in-text">Browse through the most freuently asked questions</p>
-                <form method="get">
-                    <input class="get-in-input"
+                <form action="#" method="get" class="contact-us-form">
```
Comment (per-aspera-ad-astra) [inline contact-us.html]: `method` should be `post`

**Comment by per-aspera-ad-astra** [comment]
If we start typing inside input it height will change
Screenshot 2024-08-14 at 19 30 25

File: contact-us.html [hunk 5]
```diff
@@ -120,33 +151,40 @@ <h2 class="get-in-title">
...
+                    <span class="contact-info">
+                        <input class="get-in-input name"
                         type="text"
```
Comment (per-aspera-ad-astra) [inline contact-us.html:158]: All elements inside form (except button) should have labels according to acceptance criteria

**Comment by per-aspera-ad-astra** [comment]
If search field not in focus the text becomes black:
Screenshot 2024-08-14 at 19 38 39

File: css/styles.css [hunk 6]
```diff
@@ -465,83 +659,145 @@ figure {
...
+    padding: 30px 20px;
+}
+
+.faqs-item::after {
+    content: '';
+    width: 32px;
+    height: 32px;
+    opacity: 40%;
+    background: url(../img/icons/plus+.svg);
+    background-size: cover;
```
Comment (per-aspera-ad-astra) [inline css/styles.css]: better  use `background-size: contain `in this case - `cover` can crop the image

File: css/styles.css [hunk 7]
```diff
@@ -465,83 +659,145 @@ figure {
...
+    width: 32px;
+}
+
+.faqs-list[open] .faqs-item::after  {
+    opacity: 100%;
+    background: url(../img/icons/minus-.svg);
```
Comment (per-aspera-ad-astra) [inline css/styles.css]: use `background-image` instead -  shorthand property `background` reset your `background-size`

File: css/styles.css [hunk 8]
```diff
@@ -625,95 +913,348 @@ figure {
...
+    flex-wrap: wrap;
+    gap: 12px;
+}
+
+.sidebar-list {
+    background: #141B22;
+    width: 100%;
+}
+
+.sidebar-item {
```
Comment (per-aspera-ad-astra) [inline css/styles.css:1013]: Please add `cursor: pointer` for this selector - user should be able to see that is clickable element

File: css/styles.css [hunk 9]
```diff
@@ -625,95 +913,348 @@ figure {
...
+  
+  /* Create the checkmark/indicator (hidden when not checked) */
+  .checkmark:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+  
+  /* Show the checkmark when checked */
+*.sidebar-label .sidebar-checkbox:checked ~ .checkmark:after {
```
Comment (per-aspera-ad-astra) [inline css/styles.css]: try not to use universal selector (*)

**Review by per-aspera-ad-astra — CHANGES_REQUESTED** [review]
Good work!

What I liked:

- Work meets general requirements - all forms implemented, they have validation, buttons have type attributes, forms have validation

What need to improve:

- Most information provided in comments;
- Please add labels according to this rule - [https://github.com/broacademy/HTML-course-2/blob/main/criteria.md](https://github.com/broacademy/HTML-course-2/blob/main/criteria.md) (clause 1.19)
Additionally - I see a lot of places where you use `font-weight: 400;`, I'm  sure you can remove most of them because this is default value

**Review by per-aspera-ad-astra — APPROVED** [review]
Can be merged into main branch

**Pull request merged**